src/facade/dragonfly_connection.cc

```diff
@@ -872,12 +878,18 @@ void Connection::SendMonitorMessageAsync(string msg) {
 
 void Connection::SendAsync(MessageHandle msg) {
   DCHECK(cc_);
+  DCHECK(owner());
+  DCHECK_EQ(ProactorBase::me(), owner()->socket()->proactor());
 
   if (cc_->conn_closing)
     return;
 
-  dispatch_q_bytes_.fetch_add(msg.UsedMemory(), memory_order_relaxed);
+  if (!dispatch_fb_.IsJoinable()) {
+    auto* peer = socket_.get();
+    dispatch_fb_ = MakeFiber(dfly::Launch::post, [&] { DispatchFiber(peer); });
+  }
```

Comment:
I changed the policy from `launch` to `post` because it'll start off empty and suspend immediately until I the dispatch_q is non empty -> pointless 

Alternatively we can just launch it below